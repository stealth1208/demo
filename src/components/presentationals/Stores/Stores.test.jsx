import React from 'react';
import renderer from 'react-test-renderer';
import { Stores } from '../index';

it('renders correctly', () => {
  const stores = [{
    alt: "Qoo10 Coupons & Promo Codes",
    categoryId: 0,    
    id: 917,    
    logoUrl: "https://static.shopback.com/uploads/ci/143830/logo-qoo10.gif",
    name: "Qoo10"    
  }];
    const tree = renderer
      .create(
        <Stores 
          stores={stores}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });