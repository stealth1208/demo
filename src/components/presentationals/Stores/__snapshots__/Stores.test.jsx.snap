// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="stores"
>
  <div
    className="stores__item"
  >
    <div
      className="stores__item__image"
    >
      <img
        alt="Qoo10 Coupons & Promo Codes"
        src="https://static.shopback.com/uploads/ci/143830/logo-qoo10.gif"
      />
    </div>
    <div
      className="stores__item__name"
    >
       
      Qoo10
    </div>
  </div>
</div>
`;
